package.json

```diff
@@ -115,7 +115,7 @@
     "babel-runtime": "^6.26.0",
     "backbone": "1.2.1",
     "clipboard": "1.6.1",
-    "handlebars": "4.0.13",
+    "handlebars": "4.0.14",
```

Comment:
Why not 4.1.2?

Reply:
discussed offline, we will go with 4.0.14 for now